lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
+	public function getGlobalGroups();
+
+	public function getImplicitGroups();
+
+	public function getExplicitGroups();
+
+	public function getPermissions();
+
+	public function getExplicitUserGroups( $userId );
```

Comment:
Because some function need only the id, however some other use methods defined in the user class. But you're right it can be confusing - I'll take a second look if we can standardize it, till now I simply based it on actual needs.